source/helpers/hotfix.ts

```diff
@@ -15,12 +28,17 @@ export const updateHotfixes = cache.function(async (): Promise<string[][]> => {
 		return [];
 	}
 
-	return atob(content)
-		.trim()
-		.split('\n')
-		.map(line => line.split(','));
+	const storage: HotfixStorage = [];
+	for (const [featureID, unaffectedVersion, relatedIssue] of parseCsv(atob(content))) {
+		if (featureID && relatedIssue && (!unaffectedVersion || compareVersions(unaffectedVersion, version) > 0)) {
+			storage.push([featureID as FeatureID, relatedIssue]);
```

Comment:
Is this not working? All of them are still appearing locally even if I have v21.11.1. Safari

<img width="506" alt="Screen Shot 5" src="https://user-images.githubusercontent.com/1402241/145525126-b99a6584-5798-4998-a1e4-5e8a1018dcd5.png">